content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+
+<img src="https://i.imgur.com/WGqQsqt.gif" width=800 />
+<br/>
+
+###### Hyperparameters
+If you want to see the effect of hyperparameters on the plot/performance measures, simply click on the small cog next to the learner selection. Now, the left panel is replaced by a window with all available hyperparameters – changes of these have an immediate effect of recalculating the whole model and updating all plots/numbers.
+
+<img src="https://i.imgur.com/YvODxIt.gif" width=800 />
+
+
+###### Second learner
+Only observing changes of a single learner may become a bit boring after time, so a small button is available to easily double the fun: Pressing “Add learner” gives the choice of a second learner with a second plot, making it possible to compare the prediction surfaces of two different learners on the same task right next of each other. Incredible. A bit.
+
+<img src="https://i.imgur.com/Kho9pDh.gif" width=800 />
+<br/>
+
+But wait, there’s still more to offer!
+
+###### Learning curve and ROC
+The curious user can change the shown plots on the right panel by switching to the tabs “Learning Curve” (classification and regression) or “ROC” (only classification). The former tells you how well the learner does, or the learners do compared to each other, with respect to user selected performance measurements (y-axis) on a fraction of the original train data (x-axis). Every additionally selected measure gives an additional plot. Not groundbreaking, but handy.
+
+Last but not least, the ROC curve is only an ROC curve with its default settings for the x- and y-axis: Plotting the true positive rate against the false positive rate, giving good advice of how well the learner can separate the classes (the bigger the area under the curve, the better).
+Change the settings and you’ll get quite exotic variations, whose names are beyond the scope of the author.
+
+<img src="https://i.imgur.com/qs8E51L.gif" width=800 />
+<br/>
+
+## Usage
+
+If you want to explore the mlrPlayground on your own, you have two possibilities:
+
+Either visit the website https://compstat-lmu.shinyapps.io/mlrPlayground/ (big thanks to the department of computational statistics at the LMU for hosting)
+
+or
+
+install it locally on your own machine by ``devtools::install_github("SebGruber1996/mlrPlayground")``
+and start with ``mlrPlayground::start()``.
+The first method is quicker, but may crash if too many people are using it at once, while the latter requires a long time for installation.
+<br/>
+<br/>
+
+Thanks for spending your time reading this blog post instead of being on a real playground!
+<br/> 
+<br/>
+<br/>
+
+Side note: The app is still far from perfect.
+For example the UI is -- especially for the hyperparameters -- broken for small monitor resolutions. 
```

Comment:
```suggestion
The UI is -- especially for the hyperparameters -- not working well for small displays. 
```

"small monitor resolutions" do not exist. Either "small displays" or "low resolutions",